build.gradle

```diff
@@ -147,6 +145,23 @@ project(':accurest-gradle-plugin') {
 		}
 	}
 
+	task createClasspathManifest {
+		def outputDir = file("$buildDir/$name")
+
+		inputs.files sourceSets.main.runtimeClasspath
+		outputs.dir outputDir
+
+		doLast {
+			outputDir.mkdirs()
+			file("$outputDir/plugin-classpath.txt").text = sourceSets.test.runtimeClasspath.join("\n")
+		}
+	}
+
+    // Add the classpath file to the test runtime classpath
+	dependencies {
+		testRuntime files(createClasspathManifest)
+	}
+
 	// Hack for reusing this in the plugin test
 	test.dependsOn(':accurest-core:install')
 	test.dependsOn(':accurest-messaging-root:accurest-messaging-core:install')
```

Comment:
Ah and in the src/test/resources/...messaging.../build.gradle remove the line with importing of messaging:

https://github.com/Codearte/accurest/commit/ec1751fa935b261d7eba0ee7e8c40beb2af059f0